addons/binding/org.openhab.binding.solarlog/src/main/java/org/openhab/binding/solarlog/internal/HttpUtils.java

```diff
@@ -0,0 +1,83 @@
+/**
+ * Copyright (c) 2010-2017 by the respective copyright holders.
+ *
+ * All rights reserved. This program and the accompanying materials
+ * are made available under the terms of the Eclipse Public License v1.0
+ * which accompanies this distribution, and is available at
+ * http://www.eclipse.org/legal/epl-v10.html
+ */
+
+package org.openhab.binding.solarlog.internal;
+
+import java.util.concurrent.TimeUnit;
+
+import org.eclipse.jetty.client.HttpClient;
+import org.eclipse.jetty.client.HttpResponseException;
+import org.eclipse.jetty.client.api.ContentResponse;
+import org.eclipse.jetty.client.util.StringContentProvider;
+import org.eclipse.jetty.http.HttpMethod;
+import org.eclipse.jetty.http.HttpStatus;
+
+import com.google.gson.JsonElement;
+import com.google.gson.JsonParser;
+
+/**
+ * Collection of methods to help retrieve HTTP data from a SolarLog Server
+ * Based on SqueezeBox HttpUtils, adapted for SolarLog.
```

Comment:
Are you aware of [HttpUtil in ESH](https://github.com/eclipse/smarthome/blob/3ae5d7b1b284f2bde6199bfe8dad950fd38cae39/bundles/io/org.eclipse.smarthome.io.net/src/main/java/org/eclipse/smarthome/io/net/http/HttpUtil.java)?
Could that be used? We should avoid adding such util classes everywhere, but rather consolidate what we need in ESH.